.babelrc

```diff
@@ -1,34 +1,6 @@
 {
-  "env": {
-    "development": {
-      "presets": ["es2015"],
-      "plugins": [
-        "transform-object-rest-spread",
-        "transform-async-to-generator",
-        "transform-runtime"
-      ],
-    },
-    "default": {
-      "presets": ["es2015"],
-      "plugins": [
-        "transform-object-rest-spread",
-        "transform-async-to-generator",
-        "transform-runtime"
-      ],
-    },
-    "noshim": {
-      "plugins": [
-        "transform-es2015-modules-commonjs",
-        "transform-object-rest-spread",
-        "transform-async-to-generator"
-      ]
-    },
-    "firefox": {
-      "plugins": [
-        "transform-es2015-modules-commonjs",
-        "transform-object-rest-spread",
-        "transform-async-to-generator"
-      ]
```

Comment:
we'll have to keep the `firefox` target, because we ship this library in the browser :) https://dxr.mozilla.org/mozilla-central/source/services/common/kinto-offline-client.js